Autounit (#146)
schultek/jaspr: packages/jaspr/CHANGELOG.md

## Unreleased minor

> - Had to resort for using Style.Raw for this, So I added a new Unit called Unit.auto
> - Simply emmits the value 'auto', use this for responsive styling

Reviewer: Change the wording to a more general statement, beginning with a verb. Smtg like 
```suggestion
- Added `Unit.auto` for responsive styling using the 'auto' css unit.
```
